srsly/_json_api.py

```diff
@@ -53,6 +79,42 @@ def read_json(path: FilePath) -> JSONOutput:
         return ujson.load(f)
 
 
+def read_json_dict(path: FilePath) -> Dict[str, Any]:
+    """Load JSON from file or standard input.
+
+    path (FilePath): The file path. "-" for reading from stdin.
+    RETURNS (JSONOutput): The loaded JSON content.
+    """
+    data = read_json(path)
+    if not isinstance(data, dict):
+        raise ValueError("Invalid JSON, data could not be parsed to a dict.")
+    return data
+
+
+def read_json_list(path: FilePath, validate_inner: bool = False, skip_invalid: bool = False) -> List[Dict[str, Any]]:
+    """Load JSON from file or standard input.
+
+    path (FilePath): The file path. "-" for reading from stdin.
+    RETURNS (JSONOutput): The loaded JSON content.
+    """
+
+    data = read_json(path)
+    if not isinstance(data, list):
+        raise ValueError("Invalid JSON, data could not be parsed to a list of dicts.")
```

Comment:
I think describing any of the input as "invalid JSON" is going to be confusing for users. Something like "invalid value" instead?

Reply:
Yeah good point, I changed this language so it's reads that it can't be parsed to the expected type